addons/block_code/ui/blocks/control_block/control_block.gd

```diff
@@ -144,9 +143,8 @@ func format():
 		snap_container.custom_minimum_size.y = 30
 		snap_container.size_flags_horizontal = Control.SIZE_SHRINK_BEGIN
 
-		var snap_gutter := Control.new()
+		var snap_gutter := BlockGutter.new()
```

Comment:
I think we've been trying to move away from global classes unless their really needed since it reduces the chance for startup issues with the global class cache. Instead, I think you should just preload them at the top:

```
const BlockBackground = preload("res://addons/block_code/ui/blocks/utilities/background/background.gd")
const BlockGutter = preload("res://addons/block_code/ui/blocks/utilities/background/gutter.gd")
```

Reply:
Yes, exactly. Can you make this last change @dylanmccall ?

Reply:
Ah, I missed that happening! Makes sense :) That's fixed in the newest commit.

Reply:
I think Manuel started it near the beginning of GUADEC in the context of [import errors](https://phabricator.endlessm.com/T35494) and I happened to notice it since I was trying to wrap up some of my own work that Thursday and Friday.